src/main/kotlin/org/rust/cargo/runconfig/buildtool/CargoBuildTaskRunner.kt

```diff
@@ -70,7 +70,9 @@ class CargoBuildTaskRunner : ProjectTaskRunner() {
             WaitingTask(project, waitingIndicator, queuedTask.executionStarted).queue()
         }
 
-        buildSessionsQueue.run(queuedTask, null, EmptyProgressIndicator())
+        project.service<CargoBuildSessionsQueueManager>()
```

Comment:
nit: I usually prefer creating `getInstance(project: Project)` static method in service class because:
- it's a common pattern in the platform
- it's clearer how to get the instance of this class (at least for me)

Reply:
Fixed.